### PR TITLE
Readme updates - Prereqs

### PR DESCRIPTION
**Pull request by kiritigowda**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -10,10 +10,28 @@ Radeon Performance Primitives (RPP) library is a comprehensive high-performance
...
+* [ROCm supported hardware](https://rocm.github.io/hardware.html)
+* [ROCm](https://github.com/RadeonOpenCompute/ROCm#installing-from-amd-rocm-repositories)
+* Boost
+* Clang
+* IEEE 754-based half-precision floating-point library - half.hpp
+
+**NOTE:**
+* Boost
+```
+sudo apt --yes install libblkid-dev e2fslibs-dev libboost-all-dev libaudit-dev
```
Comment (rrawther) [inline Readme.md:24]: Why does it need all these libraries in addition to libboost? I don't remember installing those

File: Readme.md [hunk 2]
```diff
@@ -10,10 +10,28 @@ Radeon Performance Primitives (RPP) library is a comprehensive high-performance
...
+```
+* CLang
+```
+sudo apt install clang
+```
+* half.hpp
+```
+wget https://raw.githubusercontent.com/ARM-software/ComputeLibrary/master/include/half/half.hpp
```
Comment (rrawther) [inline Readme.md:32]: We want to take it from this link which is the offitial half library.
http://half.sourceforge.net/

File: Readme.md [hunk 3]
```diff
@@ -10,10 +10,29 @@ Radeon Performance Primitives (RPP) library is a comprehensive high-performance
 
 RPP is developed for __Linux__ operating system.
 
-##### Prerequisites
-1. Ubuntu `16.04`/`18.04`
-2. [ROCm supported hardware](https://rocm.github.io/hardware.html)
-3. [ROCm](https://github.com/RadeonOpenCompute/ROCm#installing-from-amd-rocm-repositories)
+## Prerequisites
+* Ubuntu `16.04`/`18.04`
```
Comment (rrawther) [inline Readme.md:14]: Should we say this because there are build errors in 16.04?

Comment (kiritigowda) [reply Readme.md:14]: we have noted the bug, it might be the pre-req issue.

**Pull request merged**